### PR TITLE
ZENKO-3122 update zenko-ui version

### PR DESCRIPTION
**Pull request by nicolas2bert**

### CODE REVIEW

**Review by naren-rajendran — APPROVED** [review]

**Review by ilkescality — APPROVED** [review]

File: solution/deps.txt [hunk 1]
```diff
@@ -12,7 +12,7 @@ registry.scality.com/sf-eng/zenko-operator:0a7b2f0
 registry.scality.com/zenko-dev/backbeat:161b9c1
 registry.scality.com/zenko-dev/cloudserver:latest-8.1.2
 registry.scality.com/zenko-dev/cloudserver:4332d0a
-registry.scality.com/zenko-ui-dev/zenko-ui:2f10bac
+registry.scality.com/zenko-ui-dev/zenko-ui:0.1.0
```
Comment (miniscruff) [inline solution/deps.txt]:
```suggestion
registry.scality.com/zenko-ui/zenko-ui:0.1.0
```
I think the image is here

Comment (nicolas2bert) [reply solution/deps.txt]: Yes, thank you :)

**Review by miniscruff — APPROVED** [review]

**Comment by nicolas2bert** [comment]
@bert-e approve

**Review by tcarmet — APPROVED** [review]

**Pull request merged**